BUILDING.md

```diff
@@ -30,3 +30,65 @@ v1.1.0
 ```
 
 Note: on Windows invoke the `gfsh.bat` script to print the version string.
+
+## Setting up IntelliJ
+
+The following steps have been tested with:
+
+* **IntelliJ IDEA 2018.2.2**
+
+1. Run `./gradlew --parallel devBuild` from Geode repository root to create compiler generated source
+
+2. Import project into IntelliJ IDEA
+
+   From the **Welcome to IntelliJ IDEA** window:
+
+   1. **Import Project ->** select *build.gradle* file from Geode repository root and press **Open**.
+   2. Enable **Use auto-import**
+   3. Enable **Create separate module per source set**
+   4. Select **Use Project JDK 1.8.0_*nnn*** where *nnn* is latest build required for Geode
+   5. Expand **Global Gradle settings** and edit **Gradle VM options:** to contain `-DUSING_INTELLIJ=true` and press **OK**.
+
+   From an existing IntelliJ project (skip this if you imported from the **Welcome IntelliJ IDEA** window):
+
+   Navigate to **File -> New -> Project from Existing Sources...**, select *build.gradle* file from Geode repository root and press **Open**. Then follow b. through e. above.
```

Comment:
I am missing the referenced b-e above?

Reply:
Ack yeah this is now 2-5 with the last set of changes.  Will fix, thanks!